peanut_gb.h

```diff
@@ -1515,25 +1832,63 @@ void __gb_draw_line(struct gb_s *gb)
 				px = 0;
 				win_x = disp_x - gb->hram_io[IO_WX] + 7;
 				idx = gb->vram[win_line + (win_x >> 3)];
+#if PEANUT_FULL_GBC_SUPPORT
+				idxAtt = gb->vram[win_line + (win_x >> 3) + 0x2000];
+#endif
 
 				if(gb->hram_io[IO_LCDC] & LCDC_TILE_SELECT)
 					tile = VRAM_TILES_1 + idx * 0x10;
 				else
 					tile = VRAM_TILES_2 + ((idx + 0x80) % 0x100) * 0x10;
 
+#if PEANUT_FULL_GBC_SUPPORT
+				if (gb->cgb.cgbMode && (idxAtt & 0x08)) tile += 0x2000;  //VRAM bank 2
+				if (gb->cgb.cgbMode && (idxAtt & 0x40)) tile += 2 * (7 - py);
+				if (!gb->cgb.cgbMode || !(idxAtt & 0x40)) tile += 2 * py;
+#else
```

Comment:
`gb->cgb.cgbMode` is being check three times here when it could be reduced to one. Maybe the following could work?

```c
if(gb->cgb.cgbMode)
{
	if(idxAtt & 0x08) tile += 0x2000; //VRAM bank 2
	if(idxAtt & 0x40) tile += 2 * (7 - py);
}
else if(!(idxAtt & 0x40))
{
	tile += 2 * py;
}
```